es6-lib/decoders/geojson.js

```diff
@@ -39,14 +40,19 @@ class GeoJSON extends Transform {
     .on('data', this._onCrs.bind(this));
   }
 
-
-
   static canDecode() {
     return ['application/json'];
   }
 
   _onError(err) {
-    this.emit('error', err);
+    var parserState = this._crsParser.getError() || this._featureParser.getError() || {};
+    var [_m, reason] = /Error: ([\w ]+)/.exec(err);
```

Comment:
Is there any chance that what we're looking to capture as `reason` will contain chars not matched by `\w`?